local-db/init-pg.sql

```diff
@@ -0,0 +1,90 @@
+\connect postgres
+
+-- USER -------------------------------------------
+-- Table Definition ----------------------------------------------
+
+CREATE EXTENSION citext;
+CREATE TYPE role_type AS ENUM ('Standard', 'Admin', 'SuperAdmin');
+
+CREATE TABLE IF NOT EXISTS "user" (
+    user_id UUID PRIMARY KEY NOT NULL,
+    bceid_user_guid VARCHAR(36),
+    idir_user_guid VARCHAR(36),
+    role role_type NOT NULL, 
+    email CITEXT,
+    first_name VARCHAR(50),
+    last_name VARCHAR(75),
+    display_name VARCHAR(125),
+    CHECK (bceid_user_guid IS NOT NULL OR idir_user_guid IS NOT NULL)
+);
+
+-- ACTIVE PIN -------------------------------------------
+-- Table Definition ----------------------------------------------
+
+CREATE TYPE parcel_status_type AS ENUM ('A', 'I');
+CREATE TYPE title_status_type AS ENUM ('R', 'C');
+
+CREATE TABLE IF NOT EXISTS "active_pin" (
+    live_pin_id UUID PRIMARY KEY NOT NULL,
+    pin VARCHAR(8) NOT NULL,
```

Comment:
Based on Brett's updated requirements (recording a homeowner but not immediately issuing a PIN) this cannot be a required (NOT NULL) field